packages/components/inputs/search-select-input/src/search-select-input.js

```diff
@@ -160,6 +159,17 @@ SearchSelectInput.propTypes = {
    * Determines if option groups will be separated by a divider
    */
   showOptionGroupDivider: PropTypes.bool,
+  /**
+   * The default set of options to show before the user starts searching. When set to true, the results for loadOptions('') will be autoloaded.
+   */
+  defaultOptions: PropTypes.oneOfType([
+    PropTypes.bool,
+    PropTypes.arrayOf(
+      PropTypes.shape({
+        value: PropTypes.string.isRequired,
```

Comment:
Is this the only required field? Are there any other fields that should be provided?

Reply:
Value is required, label is kinda optional. Since if label is not provided, it will fallback to empty string but it will not break the component

Reply:
Ok but we can still include it (the `label`) so it's clear what kind of object you would need to pass.

Reply:
I have updated the search-select-input, however if we need the same change in other `select-input/fields`, we need to update their props as well as they also only require `value` as of now.